extensions/wikia/PhalanxII/fallback/Phalanx.class.php

```diff
@@ -402,10 +402,10 @@ static function isBlocked($text, $blockData, $writeStats = true) {
 	 * @author Maciej Błaszkowski <marooned at wikia-inc.com>
 	 * @author Władysław Bodzek
 	 */
-	static function findBlocked($text, $blocksData, $writeStats = true, &$matchingBlockData = null) {
+	static function findBlocked( $text, $blocksData, $writeStats = true, &$matchingBlockData = null ) {
 		wfProfileIn( __METHOD__ );
-		$result = array('blocked' => false, 'msg' => '');
-		foreach ($blocksData as $blockData) {
+		$result = array( 'blocked' => false, 'msg' => '' );
+		foreach ( $blocksData as $blockData ) {
 			if ( isset( $blockData['id'] ) ) { // full format
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)